<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sd_profile</class>
 <widget class="QDialog" name="sd_profile">
  <property name="windowModality">
   <enum>Qt::WindowModality::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>701</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(189, 201, 255);
border-raius:25px;
</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QStackedWidget" name="profiledetails">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border-radius:25px;
border-color:blue;</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="studentdetails">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QWidget" name="widget" native="true">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QWidget" name="widget_2" native="true">
            <property name="minimumSize">
             <size>
              <width>400</width>
              <height>600</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);

border-color: rgb(23, 255, 178);
border-color: rgb(85, 255, 127);
border-radius:50px;</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QWidget" name="verticalWidget_2" native="true">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>400</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">border color : black;
</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_7">
                <property name="spacing">
                 <number>10</number>
                </property>
                <item>
                 <widget class="QLabel" name="createyouraccount">
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>50</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>14</pointsize>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: rgb(0, 0, 0);</string>
                  </property>
                  <property name="text">
                   <string>Your Details</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignmentFlag::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_8">
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: rgb(0, 0, 0);
border:none;
border-radius:50px;</string>
                  </property>
                  <property name="text">
                   <string>Name:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="name_sdprofile">
                  <property name="styleSheet">
                   <string notr="true">color: rgb(0, 0, 0);
border-color: rgb(255, 84, 172);
border-radius: 15px;
background-color: rgb(253, 229, 255);</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_9">
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: rgb(0, 0, 0);</string>
                  </property>
                  <property name="text">
                   <string>Email:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="email_sdprofile">
                  <property name="styleSheet">
                   <string notr="true">color: rgb(0, 0, 0);
border-color: rgb(255, 84, 172);
border-radius: 15px;
background-color: rgb(253, 229, 255);</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_12">
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: rgb(0, 0, 0);
border:none;
border-radius:50px;</string>
                  </property>
                  <property name="text">
                   <string>ID:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="id_sdprofile">
                  <property name="styleSheet">
                   <string notr="true">color: rgb(0, 0, 0);
border-color: rgb(255, 84, 172);
border-radius: 15px;
background-color: rgb(253, 229, 255);</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_16">
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: rgb(0, 0, 0);
border:none;
border-radius:50px;</string>
                  </property>
                  <property name="text">
                   <string>Department:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="dept_sdprofile">
                  <property name="styleSheet">
                   <string notr="true">color: rgb(0, 0, 0);
border-color: rgb(255, 84, 172);
border-radius: 15px;
background-color: rgb(253, 229, 255);</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_13">
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: rgb(0, 0, 0);</string>
                  </property>
                  <property name="text">
                   <string>Class</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="class_sdprofile">
                  <property name="styleSheet">
                   <string notr="true">color: rgb(0, 0, 0);
border-color: rgb(255, 84, 172);
border-radius: 15px;
background-color: rgb(253, 229, 255);</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_14">
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: rgb(0, 0, 0);</string>
                  </property>
                  <property name="text">
                   <string>Section</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="section_sdprofile">
                  <property name="styleSheet">
                   <string notr="true">color: rgb(0, 0, 0);
border-color: rgb(255, 84, 172);
border-radius: 15px;
background-color: rgb(253, 229, 255);</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="editbutton">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>50</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(163, 0, 0);
background-color: rgb(178, 255, 255);
font: 24pt &quot;Segoe UI&quot;;
font: 700 italic 9pt &quot;Segoe UI&quot;;</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="updatedetail">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">font: 9pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 0);
font: 11pt &quot;Segoe UI&quot;;</string>
         </property>
         <property name="text">
          <string>Update your details</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>500</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(203, 255, 246);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QWidget" name="verticalWidget" native="true">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>400</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border color : black;
background-color: rgb(199, 233, 255);
</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <property name="spacing">
              <number>10</number>
             </property>
             <item>
              <widget class="QLabel" name="label_5">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>20</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(0, 0, 0);
border:none;
border-radius:50px;</string>
               </property>
               <property name="text">
                <string>Name:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="sdupdateprofile_name">
               <property name="styleSheet">
                <string notr="true">color: rgb(0, 0, 0);</string>
               </property>
               <property name="placeholderText">
                <string>Enter your full name</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_6">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>20</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(0, 0, 0);</string>
               </property>
               <property name="text">
                <string>Email:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="sdupdateprofile_email">
               <property name="styleSheet">
                <string notr="true">color: rgb(0, 0, 0);</string>
               </property>
               <property name="placeholderText">
                <string>Enter your college email</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_7">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>20</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(0, 0, 0);
border:none;
border-radius:50px;</string>
               </property>
               <property name="text">
                <string>ID:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="sdupdateprofile_id">
               <property name="styleSheet">
                <string notr="true">color: rgb(0, 0, 0);</string>
               </property>
               <property name="placeholderText">
                <string>Enter your Student Id</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_15">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>20</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(0, 0, 0);</string>
               </property>
               <property name="text">
                <string>Department</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="sdupdateprofile_dept">
               <property name="font">
                <font>
                 <bold>false</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(0, 0, 0);</string>
               </property>
               <property name="placeholderText">
                <string>Select your department</string>
               </property>
               <item>
                <property name="text">
                 <string>Science</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Commerce</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_10">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>20</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(0, 0, 0);</string>
               </property>
               <property name="text">
                <string>Class</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="sdupdateprofile_class">
               <property name="font">
                <font>
                 <bold>false</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(0, 0, 0);</string>
               </property>
               <property name="placeholderText">
                <string>Select your class</string>
               </property>
               <item>
                <property name="text">
                 <string>11</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>12</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_11">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>20</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(0, 0, 0);</string>
               </property>
               <property name="text">
                <string>Section</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="sdupdateprofile_section">
               <property name="font">
                <font>
                 <bold>false</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(0, 0, 0);</string>
               </property>
               <property name="placeholderText">
                <string>Select your section</string>
               </property>
               <item>
                <property name="text">
                 <string>A</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>B</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="backButton">
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);</string>
            </property>
            <property name="text">
             <string>Back</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="savebutton">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>50</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);
font: 600 9pt &quot;Segoe UI&quot;;
background-color: rgb(255, 133, 149);</string>
            </property>
            <property name="text">
             <string>Save</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="cancelbutton">
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);</string>
            </property>
            <property name="text">
             <string>Cancel</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
